components/meter/README.md

```diff
@@ -0,0 +1,49 @@
+# Meter Components
+
+* Note this is considered as progress indicator in design.
+
+## d2l-meter-linear
+<img src="../../screenshots/meter/d2l-meter-linear-no-progress.png">
+<img src="../../screenshots/meter/d2l-meter-linear-has-progress.png">
+<img src="../../screenshots/meter/d2l-meter-linear-completed.png">
```

Comment:
I don't think this will work. You may need to do:
`![description of image](../../screenshots/d2l-meter-linear-no-progress.png?raw=true)`

Alternately I don't think I'd mind if the screenshots lived right here in this directory. *shrug*

Reply:
It did work but I can do this instead. Seems more markup.

Reply:
The reason I've used `img` in the past instead of the `![description of image](../../screenshots/d2l-meter-linear-no-progress.png?raw=true)` way was because I wanted to make the width smaller without having to shrink the actual image.  I wasn't sure how to do this with the second way.